themes/digital.gov/layouts/partials/core/head.html

```diff
@@ -121,6 +121,8 @@
   <meta property="og:site_name" content="Digital.gov" />
```

Comment:
For the robots metadata we could add:

```
<meta name="robots" content="index, follow">
```
which tells search engines to index and follow the current page.

Reply:
added to file